utils/lpac/files/lpac.sh

```diff
@@ -8,26 +8,29 @@ APDU_DEBUG="$(uci_get lpac global apdu_debug 0)"
 HTTP_BACKEND="$(uci_get lpac global http_backend curl)"
 HTTP_DEBUG="$(uci_get lpac global http_debug 0)"
 
-export LPAC_HTTP="$HTTP_BACKEND"
+function export_if_not_in_env {
+    eval "value=\${$1}"
+    if [ -z "$value" ]; then
+        export "$1=$2"
+    fi
+}
```

Comment:
IMO, the function is overkill, see my approach which goes without:
https://github.com/openwrt/packages/pull/25046/files

Reply:
I wouldn't name it "a clean way", sorry.